### PR TITLE
Added is_global option to set global plot extent

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

File: geoviews/plotting/bokeh/plot.py [hunk 1]
```diff
@@ -96,6 +99,8 @@ def get_extents(self, element, ranges):
         set_extent method to project the extents to the
         Elements coordinate reference system.
         """
+        if self.is_global:
+            return (-20026376.39, -20048966.10, 20026376.39, 20048966.10)
```
Comment (jbednar) [inline geoviews/plotting/bokeh/plot.py]: I don't quite get the reason for the is_global flag (some description in the PR would be useful!), but in any case, how does knowing it's a plot of the whole globe determine the range here?  Can't there be infinitely many plots of the whole globe, centered at different longitudes?

Comment (philippjfr) [reply geoviews/plotting/bokeh/plot.py]: It's modeled on the cartopy ``ax.set_global()`` method, which simply means "Set the extent of the Axes to the limits of the projection."

>Can't there be infinitely many plots of the whole globe, centered at different longitudes?

No, neither matplotlib nor currently bokeh support wrapping at arbitrary longitudes. I have no issue adding a ``central_longitude`` option at a later point but that will require a whole bunch more work (and I'm not yet sure that it's even possible for certain projections in matplotlib).

Comment (philippjfr) [reply geoviews/plotting/bokeh/plot.py]: Actually I take the part about matplotlib back, it is possible to set the ``central_longitude`` in matplotlib directly on the ``crs`` in some cases.

**Comment by jlstevens** [comment]
Just as a matter of taste, I think ``is_global`` is a pretty horrible name that doesn't really convey what it does. Why not just ``global`` or ``globe``?

**Comment by philippjfr** [comment]
I don't particularly disagree, ``globe`` is imo wrong, and global cannot be used because ``global = ...`` produces a syntax error.

**Comment by jlstevens** [comment]
Why do you think ``globe`` is wrong? Maybe I don't know exactly what this parameter is doing but it *sounds* like it should you the whole globe (assuming the extents of projections go that far?). To be honest, something like ``show_globe`` or ``show_global`` might be more consistent with holoviews (though I'm not such a fan of these ``show_X`` methods).

**Comment by philippjfr** [comment]
globe in the context of cartopy refers specifically to projections that actually render the earth as a globe, which means that part of it is obscured. Not a fan of ``show_...`` either but ``show_global`` is probably the best option here.

**Comment by philippjfr** [comment]
Maybe ``global_extent``?

**Pull request merged**